src/views/catalog/wizard/tabs/overview/WizardOverviewTab.tsx

```diff
@@ -114,6 +115,13 @@ const WizardOverviewTab: WizardTab = ({ vm, tabsData, updateVM }) => {
               }
             />
 
+            <WizardDescriptionItem
+              className="wizard-overview-description-left-column"
+              title={t('Machine type')}
+              testId="wizard-overview-machine-type"
+              description={getMachineType(vm) || NO_DATA_DASH}
```

Comment:
no need for `NO_DATA_DASH`
`WizardDescriptionItem` will show `Not available` if `description` is falsy